test/powershell/Modules/Microsoft.PowerShell.Utility/xml.tests.ps1

```diff
@@ -55,19 +55,31 @@
         }
 
         It "non filesystem path using <parameter>" -TestCases @(
-            @{parameter="literalPath"},
-            @{parameter="path"}
+            @{parameter="literalPath"; expectedError='ProcessingFile,Microsoft.PowerShell.Commands.SelectXmlCommand'},
+            @{parameter="path"; expectedError='ProcessingFile,Microsoft.PowerShell.Commands.SelectXmlCommand'}
```

Comment:
Could you please even  `expectedError` ?

Reply:
@iSazonov: I understand what you're asking.

Reply:
@iSazonov is referring to formatting to align the parameters making it easier to read

Reply:
Sure, I'll reformat them.